tests/acceptance/pageObjects/FilesPageElement/versionsDialog.js

```diff
@@ -0,0 +1,49 @@
+module.exports = {
+  commands: {
+    getVersions: async function (filename) {
```

Comment:
that should not depend on the filename, the versions dialog PO should not have to open itself

Reply:
done!